linux/client_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package linux
+
+import (
+	"bytes"
+	"crypto"
+	"testing"
+)
+
+var testModule = "/usr/local/lib/softhsm/libsofthsm2.so"
+var testSlot = "0x268c8a20"
+var testLabel = "Demo Object"
+var testUserPin = "0000"
```

Comment:
Should these be const?

Reply:
I think they should be const. I left it as var originally because I thought the filepath might vary based on your comments on my previous PR

Reply:
I updated these and the vars in pkcs11_test.go to be const